suisa_sendemeldung/suisa_sendemeldung.py

```diff
@@ -374,9 +374,12 @@ def merge_duplicates(data):
 def funge_release_date(release_date: str = ""):
     """Make a release_date from ACR conform to what seems to be the spec."""
 
-    if len(release_date) == 10:
-        # we can make it look like what suisa has in their examples if it's the right length
-        return datetime.strptime(release_date, "%Y-%m-%d").strftime("%Y%m%d")
+    try:
+        if len(release_date) == 10:
+            # we can make it look like what suisa has in their examples if it's the right length
+            return datetime.strptime(release_date, "%Y-%m-%d").strftime("%Y%m%d")
+    except ValueError:
+        return ""
```

Comment:
It might be more straight forward to just filter the invalid example:
```suggestion
    if len(release_date) == 10 and release_date != "0000-00-00":
        # we can make it look like what suisa has in their examples if it's the right length
        return datetime.strptime(release_date, "%Y-%m-%d").strftime("%Y%m%d")
```
or maybe it makes more sense to have the `try...except` in the `if`?
```suggestion
    if len(release_date) == 10:
        try:
            # we can make it look like what suisa has in their examples if it's the right length
            return datetime.strptime(release_date, "%Y-%m-%d").strftime("%Y%m%d")
        except ValueError:
            return ""
```
I also took a look at my e2e+typing pr: https://github.com/radiorabe/python-acrclient/pull/16 but i only have checking for a string in there for now (validating dates and whatnot is still todo there).

Reply:
Yes putting the try in the if makes more sense.